Add `failureIssueThreshold` and `recoveryThreshold` to js cron docs (#10168)
getsentry/sentry-docs: includes/javascript-crons-upsert.mdx

`timezone`:

The `tz` where your job is running. This is usually your server's timezone, (such as `America/Los_Angeles`). See [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Optional.

`failureIssueThreshold`:

> _requires SDK version `8.6.0` or higher_

Author: ```suggestion
_requires SDK version `8.7.0` or higher_
```

> The number of consecutive failed check-ins it takes before an issue is created. Optional.

Reviewer: ```suggestion
The number of consecutive failed check-ins required before an issue can be created. Optional.
```

`recoveryThreshold`:

> _requires SDK version `8.6.0` or higher_

Author: ```suggestion
_requires SDK version `8.7.0` or higher_
```

> The number of consecutive OK check-ins it takes before an issue is resolved. Optional.

Reviewer: ```suggestion
The number of consecutive successful check-ins required for an issue to be considered resolved. Optional.
```
